climada/util/coordinates.py

```diff
@@ -2646,36 +2646,21 @@ def set_df_geometry_points(df_val, scheduler=None, crs=None):
     ----------
     df_val : GeoDataFrame
         contains latitude and longitude columns
-    scheduler : str, optional
-        used for dask map_partitions. “threads”, “synchronous” or “processes”
+    scheduler : str
+        deprecated, obsolete
     crs : object (anything readable by pyproj4.CRS.from_user_input), optional
         Coordinate Reference System, if omitted or None: df_val.geometry.crs
     """
     LOGGER.info('Setting geometry points.')
+    if scheduler is not None:
+        LOGGER.warn("The scheduler argument is obsolete, dask support has been removed from the"
```

Comment:
`warn` is deprecated. I also make this mistake frequently 😅 
```suggestion
        LOGGER.warning("The scheduler argument is obsolete, dask support has been removed from the"
```

Reply:
yeah I know, one of the two is deprecated, I just never know which one and counted on the IDE linter. tsh.